game-server/data/handlers/ai/instance/empyreanCrucible/SparkOfDarknessAI.java

```diff
@@ -28,11 +27,16 @@ protected void handleSpawned() {
 	private void startEventTask() {
 		ThreadPoolManager.getInstance().schedule(() -> {
 			if (!isDead())
-				SkillEngine.getInstance().getSkill(getOwner(), 19554, 1, getOwner()).useNoAnimationSkill();
+				getOwner().queueSkill(19554, 1);
```

Comment:
Queueing a skill on spawn does not cast it immediately, and only if the npc has aggro towards a target in range.
Therefore skills cast on spawn should not use `queueSkill`.